tools.mk

```diff
@@ -38,7 +38,7 @@ $(CONVERSION_GEN): ## Build conversion-gen from tools folder.
 
 $(GOLANGCI_LINT): ## Build golangci-lint from tools folder.
 	echo $(ROOT_DIR)
-	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/golangci/golangci-lint/cmd/golangci-lint@v1.26.0
+	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/golangci/golangci-lint/cmd/golangci-lint@v1.29.0
```

Comment:
these version numbers should probably be in variables

Reply:
They _kindof_ are because `GOLANGCI_LINT` this is a target? They're not used outside of the context of this target.

Gonna leave this as is for now. Can always fix later.